### ISSUE TITLE
A couple of questions on Convert parameters 

### ISSUE THREAD

**Issue by jondubin** [deepfakes/faceswap#133]
Hey! Appreciate all the work everyone's been doing on this repo. I've had a couple of questions about some of the inner workings of the convert phase. Happy to translate any answers into documentation.

- First of all, what's the difference between Convert Adjust and Convert Masked? I understand that Masked creates a new face and overlays it over the target face. But what does Adjust do? When I've tried Adjust, I've gotten some pretty distorted results, so I'm curious how they differ.

- Additionally, what do the two parameters "smooth mask" and "avg color adjust" do for Adjust and why do they only apply to Adjust, not Convert?

- Finally, what do "seamless", "erosion kernel", and "blur size" do for Masked? I originally thought blur blurred the edges, but it actually seems like it blurs the whole mask. Is it "erosion kernel" that's eroding the edges?

**Comment by Clorr** [comment]
All the plugins have a description at the beginning to tell where it does come from. The Masked and Align come from early modifications of the original script which I took as they came out. (I should retrieve exact links, the links in the files are just general indications)

FYI, the original script was quite awful with squared faces and strange skin tones. There were some attempt to get it better but with quite a big overhead. Some guy came with a really simple simple solution, which is the "Adjust" version. Finally, the most successful version was one with face detection and landmarks. But it was adding `pytorch` as dependency, and I just removed that to use `face_recognition` lib which has all what's need and is quite fast and lightweight

For all your questions, I encourage you to go on reddit. If you get your answers, feel free to add a heading in the plugin files with a documentation and a how-to, this would be extra nice ;-)
